console/src/components/Services/ApiExplorer/Analyzer/AnalyzeText.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+type ExplainPayload = {
+  field: string;
+  sql: string;
+  plan: string[];
+};
+
+type RootFieldsProps = {
+  data: ExplainPayload[];
+  activeNode: number;
+  onClick: (e: React.MouseEvent<HTMLDivElement>) => void;
+};
+
+const AnalyzeText: React.FC<RootFieldsProps> = ({
+  data,
+  activeNode,
+  onClick,
+}) => (
+  <ul>
+    {data.map((analysis, i) => {
+      return (
+        analysis.field && (
+          <div
```

Comment:
We need to change it back to `li` as it's displayed as a list. It would require you to change onClick type to: 
`onClick: (e: React.MouseEvent<HTMLLIElement>) => void;`

<img width="131" alt="Screenshot 2020-06-04 at 20 31 53" src="https://user-images.githubusercontent.com/9019397/83797250-a6b4c900-a6a2-11ea-925e-95f2e8489468.png">